### ISSUE TITLE
Definition list with no children or text causes NPE

### ISSUE THREAD

**Issue by ysb33r** [asciidoctor/asciidoctorj#467]
Just having this
```
Operating System::
```
without any children nodes or description text will lead to a NPE in asciidoctorj. A typical stacktrace will look like

```
at org.asciidoctor.ast.NodeConverter$NodeType.isInstance(NodeConverter.java:112)
at org.asciidoctor.ast.NodeConverter$NodeType$3.isInstance(NodeConverter.java:61)
at org.asciidoctor.ast.NodeConverter$NodeType.getNodeType(NodeConverter.java:100)
at org.asciidoctor.ast.NodeConverter$NodeType.access$100(NodeConverter.java:28)
at org.asciidoctor.ast.NodeConverter.createASTNode(NodeConverter.java:139)
at org.asciidoctor.internal.RubyBlockListDecorator.get(RubyBlockListDecorator.java:84)
```

This might be related to asciidoctor/asciidoctor#1789.

**Comment by robertpanzer** [comment]
Fixed with #468.
Unfortunately the build result is not pushed to the snapshot repo as the upstream build currently fails.

**Comment by robertpanzer** [comment]
Fixed in 1.6.0-alpha.3
